app/src/main/res/layout/fragment_shop_update.xml

```diff
@@ -164,6 +163,94 @@
+                    android:text="@string/avatar"
+                    android:textColor="@color/color_black"
+                    android:textStyle="bold"
+                    />
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    >
+                    <ImageView
+                        android:id="@+id/image_avatar"
+                        android:layout_width="@dimen/dp_100"
+                        android:layout_height="@dimen/dp_100"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:background="@drawable/border_layout"
+                        android:backgroundTint="@color/color_gray_white"
+                        android:padding="@dimen/dp_5"
+                        android:scaleType="centerCrop"
+                        app:imageUrl="@{viewModel.imageAvatar}"
+                        tools:ignore="ContentDescription,UnusedAttribute"
+                        />
+                    <ImageButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerVertical="true"
+                        android:layout_gravity="center"
+                        android:layout_marginLeft="@dimen/dp_20"
+                        android:layout_marginStart="@dimen/dp_20"
+                        android:layout_toEndOf="@+id/image_avatar"
+                        android:layout_toRightOf="@+id/image_avatar"
+                        android:contentDescription="@string/avatar"
+                        android:onClick="@{()-> viewModel.onClickChooseAvatar()}"
+                        app:srcCompat="@drawable/ic_add_image"
+                        />
+                </RelativeLayout>
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/dp_20"
+                    android:gravity="center"
+                    >
+                    <ImageView
+                        android:id="@+id/image_cover"
+                        android:layout_width="@dimen/dp_250"
+                        android:layout_height="@dimen/dp_150"
+                        android:layout_marginLeft="@dimen/dp_10"
+                        android:layout_marginStart="@dimen/dp_10"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:background="@drawable/border_layout"
+                        android:backgroundTint="@color/color_gray_white"
+                        android:contentDescription="@string/cover_image"
+                        android:padding="@dimen/dp_5"
+                        android:paddingEnd="@dimen/dp_5"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingEnd` you should probably also define `paddingStart` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.